docs/style-examples.csv

```diff
@@ -0,0 +1,11 @@
+Style, Intended use, Markdown you enter like this,  Will be rendered like this
+
+Emphasis, Keywords on the first use and important phrases, `**HTML in Markdown is discouraged**`, **HTML in Markdown is discouraged**
+
+Literal text, Text that is entered displayed or processed, `<code>ACGT</code>`<br><code>\\`ACGT\\`</code><br><code>\\`NC_000023.11:g.32849790T>A\\`</code>, <code>ACGT</code><br>`ACGT`<br>`NC_000023.11:g.32849790T>A`
+
+Literal text with placeholders, Conceptual references within literal text, <code>The syntax is \\`REFERENCE:TYPE.POSEDIT\\`</code>,  The syntax is `REFERENCE:TYPE.POSEDIT`
+
+Sequence "spots", Highlight spans of sequence for discussion or emphasis, `The 3rd proline in <code>ProPro<code class="ins">Pro</code></code>`<br><code>Deletion of \`T\` in &lt;code>CAGCAG&lt;code class="del">T&lt;/code>CAG&lt;/code></code>, The 3rd proline in <code>ProPro<code class="ins">Pro</code></code><br>Deletion of `T` in <code>CAGCAG<code class="del">T</code>CAG</code> 
+
+Links, HTML links to other documents or sections, `[General Recommendations](/recommendations/general/)`<br>`[Heading Title](#headingtag)`, [General Recommendations](/recommendations/general/)<br>[Heading Title](#headingtag)
```

Comment:
```suggestion
Links, HTML links to other documents or sections, `[General Recommendations](/recommendations/general/)`<br>`[Heading Title](#headingtag)`, [General Recommendations](/recommendations/general/)<br><br>[Heading Title](#headingtag)
```

Reply:
As with the other <br> suggestion, this doesn't work reliably.